web/viewer.css

```
@@ -1264,6 +1223,47 @@ dialog :link {
 }
 
 #secondaryToolbar {
+  .toolbarButton {
+    border-radius: 0;
+    display: inline-block;
+    height: auto;
+    margin: 0;
+    padding: 0 0 1px;
+    padding-inline-start: 36px;
+    position: relative;
+    min-height: 26px;
+    min-width: 100%;
+    text-align: start;
+    white-space: normal;
+    width: auto;
+
+    &::before {
+      inset-inline-start: 12px;
+      opacity: var(--doorhanger-icon-opacity);
+      top: 5px;
+    }
+
+    &:not(.toggled):is(:hover, :focus-visible) {
+      background-color: var(--doorhanger-hover-bg-color);
+      color: var(--doorhanger-hover-color);
+    }
+
+    > span {
+      display: unset;
+      padding-inline-end: 4px;
+    }
+  }
+
+  a.toolbarButton {
```

Nit: Could this be shortened?
```suggestion
  &.a {
```

---

Or even moved into the `.toolbarButton` block above?